doc-Infrastructure_Migration_Solution_Guide/cfme/doc/modules/proc_Uploading_the_virtio_and_guest_tool_image_files_to_the_storage_domain.adoc

```diff
@@ -0,0 +1,22 @@
+[id="Uploading_the_virtio_and_guest_tool_image_files_to_the_storage_domain"]
```

Comment:
Can this file be deleted now that we're using links to the RHV content?

Reply:
Done